server/database/models/ClientRepository.js

```diff
@@ -0,0 +1,78 @@
+const AbstractRepository = require("./AbstractRepository");
+
+class ClientRepository extends AbstractRepository {
+  constructor() {
+    // Call the constructor of the parent class (AbstractRepository)
+    // and pass the table name "client" as configuration
+    super({ table: "clients" });
+  }
+
+  // The C of CRUD - Create operation
+
+  async create(client) {
+    // Execute the SQL INSERT query to add a new client to the "clients" table
+    const [result] = await this.database.query(
+      `insert into ${this.table} (Email, Password,FirstName,LastName,NumberPhone) values (?, ?, ?, ?, ?)`,
+      [
+        client.Email,
+        client.Password,
+        client.FirstName,
+        client.LastName,
+        client.NumberPhone,
+      ]
+    );
+
+    // Return the ID of the newly inserted client
+    return result.insertId;
+  }
+
+  // The Rs of CRUD - Read operations
+
+  async read(id) {
+    // Execute the SQL SELECT query to retrieve a specific client by its ID
+    const [rows] = await this.database.query(
+      `select * from ${this.table} where Id_Client = ?`,
```

Comment:
field names are in snake case, and id is sufficient for the id field, which is the primary key